content/docs/orgs/restricting-users-to-trusted-ip-ranges.md

```diff
@@ -0,0 +1,20 @@
+---
+menu:
+  docs:
+    parent: orgs
+title: Restricting users to trusted IP ranges
+---
+
+You can configure cloud.gov to restrict users in a particular domain (e.g., `agency.gov`) from using cloud.gov services unless their requests originate from a trusted network location. You can use this feature to help your agency comply with the [Trusted Internet Connections (TIC) Initiative](https://www.dhs.gov/trusted-internet-connections) while using cloud.gov. 
+
+This feature only limits requests to cloud.gov services, such as the cloud.gov API accessed via the command line and the [cloud.gov dashboard](https://dashboard.fr.cloud.gov). This feature does not restrict requests directly to applications that you run on cloud.gov, though [you can do that](#restricting-access-to-your-own-applications) on your own.
+
+### Contact support to configure restrictions for your domain
+Create a [support ticket](mailto:cloud-gov-support@gsa.gov?body=Email%20domain%3A%0A%0AEgress%20IP%20ranges%3A%0A%0AAgency%20CIO%3A%0A) specifying the IP address ranges that are valid for your domain. Because address restriction applies to all cloud.gov users from your email domain, we will request confirmation from your agency CIO before changing the configuration.
```

Comment:
@mogul Should we have a short note here that this isn't available _yet_ but will be available in the future?